docs/reference/components-privileges.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Components Privileges
+---
+
+### karmada-operator
+```yaml
+rules:
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases # karmada-operator requires access to the Lease resource for leader election.
+    verbs:
```

Comment:
RBAC permission configuration for the component can be displayed in a table.

Reply:
Updated.